spec/manageiq

```diff
@@ -0,0 +1 @@
+/home/agrare/src/manageiq/manageiq
```

Comment:
probably shouldn't be here.

Reply:
Ugh yeah, the .gitignore for this repo is all wrong

Reply:
Fixed